docs/specs/shapes.md

```diff
@@ -7,6 +7,37 @@ The graphical elements are divided in 4 categories:
 * [Styles](#shape-style), that define the visual appearance of shapes
 * [Modifiers](#modifier) alter the curves of the shapes
 
+## Shape Rendering Model
+
+The Lottie shape rendering model follows [AfterEffects' Shape Layer](
+https://helpx.adobe.com/after-effects/using/overview-shape-layers-paths-vector.html) semantics.
+
+### Grouping and Ordering Rules
+
+* **shapes** are rendered in reverse order (bottom->top)
+* **groups** offer a scoping mechanism for transforms, styles, and modifiers
+* **transforms** apply to all elements within scope, including group-nested elements
```

Comment:
Perhaps it's better to mention that the transform changes the parent group's coordinate system